site/_resources/upgrading.md

```diff
@@ -29,6 +29,52 @@ Please see the [Envoy Release Notes][17] for information about issues fixed in E
 
 ## The easy way to upgrade
 
+If the following are true for you:
+
+ * Your installation is in the `projectcontour` namespace.
+ * You are using our [quickstart example][18] deployments.
+ * Your cluster can take few minutes of downtime.
+
+Then the simplest way to upgrade to 1.4.0 is to delete the `projectcontour` namespace and reapply one of the example configurations:
+
+```
+$ kubectl delete namespace projectcontour
+$ kubectl apply -f https://projectcontour.io/quickstart/contour.yaml
+```
+
+This will remove both the Envoy and Contour pods from your cluster and recreate them with the updated configuration.
+If you're using a `LoadBalancer` Service, (which most of the examples do) deleting and recreating may change the public IP assigned by your cloud provider.
+You'll need to re-check where your DNS names are pointing as well, using [Get your hostname or IP address][12].
+
+**Note:** If you deployed Contour into a different namespace than `projectcontour` with a standard example, please delete that namespace.
+Then in your editor of choice do a search and replace for `projectcontour` and replace it with your preferred name space and apply the updated manifest.
+
+## The less easy way
+
+This section contains information for administrators who wish to apply the Contour 1.3.0 to 1.4.0 changes manually.
+
+### Upgrade to Contour 1.4.0
+
+Change the Contour image version to `docker.io/projectcontour/contour:v1.4.0`
+
+Reapply the Contour CRDs and RBAC. From within a clone of the repo, checkout `release-1.4`, then you can:
+```
+kubectl apply -f kubectl apply -f examples/contour/01-crds.yaml
+kubectl apply -f examples/contour/02-rbac.yaml
```

Comment:
You also need examples/contour/00-common.yaml (envoy service account) and examples/contour/03-envoy.yaml (service account and envoy version)

Reply:
Thanks.